### PR TITLE
[PE-4851] Backstop test for Activity component

### PR DESCRIPTION
**Pull request by sabah1910**
@jllr @janibolkvadze @mattnickles https://ctl.atlassian.net/browse/FSPE-4851

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://51608e11a641c8f919e0a83643fc2d606f935d58-chi-pr.enterprise.ctl.io" target="_blank"> https://51608e11a641c8f919e0a83643fc2d606f935d58-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1201/ for more details.

File: src/custom-elements/src/components/date/date.tsx [hunk 1]
```diff
@@ -150,7 +150,7 @@ export class Date {
   };
 
   private _initCalendarViewModel(): void {
-    dayjs.locale(this.locale);
+    // dayjs.locale(this.locale);
```
Comment (janibolkvadze) [inline src/custom-elements/src/components/date/date.tsx:153]: @sabah1910 Please uncomment this line and Rebase from master

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://cadaec8893f95fa3c0c486f6abb574388fa00f1c-chi-pr.enterprise.ctl.io" target="_blank"> https://cadaec8893f95fa3c0c486f6abb574388fa00f1c-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1227/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://4ca40b7f88f83da09aad76b961a0725aebea7119-chi-pr.enterprise.ctl.io" target="_blank"> https://4ca40b7f88f83da09aad76b961a0725aebea7119-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1228/ for more details.

File: src/custom-elements/src/components/date/date.tsx (hunk 1 above)
Comment (sabah1910) [reply src/custom-elements/src/components/date/date.tsx]: @janibolkvadze Done

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://aa9dc79a2086ad5f87fe21ae9ea0fe332b262ec6-chi-pr.enterprise.ctl.io" target="_blank"> https://aa9dc79a2086ad5f87fe21ae9ea0fe332b262ec6-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1239/ for more details.

**Comment by mattnickles** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1243/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://a7673782fdea01c55fbc46326dcfb34643e22199-chi-pr.enterprise.ctl.io" target="_blank"> https://a7673782fdea01c55fbc46326dcfb34643e22199-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1249/ for more details.

File: test/chi/components/activity.pug [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+                  i.chi-icon.-icon--muted.icon-circle-plus-outline
+                  div
+                    div Updating machine IPs
+                    .-time 3 minutes ago
+            .chi-activity__end-footer 04/21/2019
+
+
+
+
+
```
[the remark this answers is not in the document]
Comment (sabah1910) [reply test/chi/components/activity.pug]: @mattnickles Done

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-hyn1uc

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-u4mwrw, https://ops.elasticbox.com/services/instances/i-a0zart, https://ops.elasticbox.com/services/instances/i-0ai2u1, https://ops.elasticbox.com/services/instances/i-chaonx

**Pull request merged**